lib/_stream_writable.js

```diff
@@ -760,3 +760,7 @@ Writable.prototype._undestroy = destroyImpl.undestroy;
 Writable.prototype._destroy = function(err, cb) {
   cb(err);
 };
+
+Writable.prototype[Symbol.for('nodejs.rejection')] = function(err) {
```

Comment:
nit: perhaps make an internal utility constant somewhere for `Symbol.for('nodejs.rejection')`?

Reply:
how would you name it? `events.captureRejectionSym`?

Reply:
Yes, but full name... `events.captureRejectionSymbol`